### PR TITLE
Better debugger panel

### PR DESCRIPTION
**Pull request by benrudolph**
@biyeun @wpride this fixes some issues with the debugger on app preview. due to the various sizes the debugger can be, it unfortunately requires javascript to control the sizing.

screen shot 2016-11-26 at 6 57 49 pm
screen shot 2016-11-26 at 6 57 39 pm

🐟 or 🏠 
cc: @mkangia

### CODE REVIEW

File: corehq/apps/preview_app/templates/preview_app/base.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% load hq_shared_tags %}
 {% load compress %}
 {% load statici18n %}
+<!DOCTYPE html>
```
Comment (benrudolph) [inline corehq/apps/preview_app/templates/preview_app/base.html]: turns out lots of crazy css things happen when you don't specify this

File: corehq/apps/cloudcare/templates/formplayer/dependencies.html [hunk 2]
```diff
@@ -11,8 +11,8 @@
 <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.js' %}"></script>
 <script src="{% static 'datetimepicker/build/jquery.datetimepicker.full.js' %}"></script>
 <script src="{% static 'cloudcare/js/util.js' %}"></script>
-<script src="{% static 'Caret.js/dist/jquery.caret.min.js' %}"></script>
-<script src="{% static 'At.js/dist/js/jquery.atwho.min.js' %}"></script>
+<script src="{% static 'Caret.js/dist/jquery.caret.js' %}"></script>
+<script src="{% static 'At.js/dist/js/jquery.atwho.js' %}"></script>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/formplayer/dependencies.html]: had to do a fair bit of debugging these libraries to get atwho to work correctly in an iframe. these will get compressed later anyways

**Comment by benrudolph** [comment]
ok this is g2g

**Comment by wpride** [comment]
looks good to me but will let @biyeun merge

**Pull request merged**